src/middlewared/middlewared/plugins/cluster_linux/ctdb_shared_volume.py

```diff
@@ -196,6 +193,7 @@ def mount(self, job):
         else:
             mounted = True
 
+        self.middleware.call_sync('service.start', 'glustereventsd')
```

Comment:
Will we benefit from try/finally block to ensure that glustereventsd is always started again despite any (now seemingly impossible, but still) error?

Reply:
Good idea!